aim/engine/configs.py

```diff
@@ -16,6 +16,7 @@
 AIM_PARAMS_DIR_NAME = 'hyperparameters'
 AIM_MAP_DIR_NAME = 'map'
 AIM_STATS_DIR_NAME = 'stats'
+AIM_TEXT_DIR_NAME = 'document_text'
```

Comment:
Lets have directory names in plural form, cause 1 directory can store a collection of several artifacts